src/JwksHandler.ts

```diff
@@ -0,0 +1,120 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { Constants } from "./utils/Constants";
+import { Jwk, JWKSBody, Algorithm } from "./utils/IVeriCredential";
+import { PutObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import { NodeHttpHandler } from "@aws-sdk/node-http-handler";
+import crypto from "crypto";
+import * as AWS from "@aws-sdk/client-kms";
+import { EnvironmentVariables } from "./services/EnvironmentVariables";
+import { ServicesEnum } from "./models/enums/ServicesEnum";
+
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : "DEBUG";
+const POWERTOOLS_SERVICE_NAME = process.env.POWERTOOLS_SERVICE_NAME ? process.env.POWERTOOLS_SERVICE_NAME : Constants.JWKS_LOGGER_SVC_NAME;
+const logger = new Logger({
+	logLevel: POWERTOOLS_LOG_LEVEL,
+	serviceName: POWERTOOLS_SERVICE_NAME,
+});
+
+const s3Client = new S3Client({
+	region: process.env.REGION,
+	maxAttempts: 2,
+	requestHandler: new NodeHttpHandler({
+		connectionTimeout: 29000,
+		socketTimeout: 29000,
+	}),
+});
+const environmentVariables: EnvironmentVariables = new EnvironmentVariables(logger, ServicesEnum.JWKS_SERVICE);
+
+const kmsClient = new AWS.KMS({
+	region: process.env.REGION,
+});
+
+class JwksHandler implements LambdaInterface {
```

Comment:
There are some unit tests for this in CIC, could we bring those over too to give us a little coverage boost?